_pages/openacr.md

```diff
@@ -5,10 +5,10 @@ sidenav: false
 permalink: /openacrs/
 ---
 
-# Open Accessibility Conformance Reports
+# **{{page.title}}**
 
 To support the needs of customers and staff with disabilities, organizations need to know what barriers their Information and communications technology (ICT) has. One of the key goals OpenACR was to make listing ACR's easier. It should be simple for everyone involved in procurement within an organization to view the available accessibility documentation.
 
-Having a central location to store and share ACRs within an organization should reduce the overhead involved in simply collecting these reports from vendors. If agencies create their own ACRs for ICT that they use, these could be easily included in this directory as well.
+Having a central location to store and share ACRs within an organization should reduce the overhead involved in simply collecting these reports from vendors. If agencies create their own ACRs for ICT that they use, these could be included in this directory as well.
 
 The existing ACR collection can be found in the following [data table](../openacr).
```

Comment:
This relative one will need to be changed to `../openacr/` as well.

Reply:
Changed.